bedita-app/controllers/api_base_controller.php

```diff
@@ -2125,6 +2124,64 @@ protected function postFiles($objectType = null, $fileName = null) {
         ));
     }
 
+    /**
+     * set child priority and update siblings priority
+     *
+     * @param {String} $objectId
+     * @param {String} $childId
+     * @param {Number} $priority
+
```

Comment:
Missing `@return` from annotataion and `@param` annotations are incompleted and seems to have a javascript like style